src/open_inwoner/components/templates/components/Modal/Modal.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n icon_tags button_tags %}
+
+<div class="modal" id="modal" aria-labelledby="modal__title" aria-describedby="modal__text">
```

Comment:
If `id` is the same as `class`, how can this help to differentiate one modal from others?

Reply:
It will differentiate because this addClass will only happen on click on this very individual node.